pkg/api/componentreadiness/middleware/regressiontracker/regressiontracker.go

```diff
@@ -118,6 +118,10 @@ func (r *RegressionTracker) PostAnalysis(testKey crtest.Identification, testStat
 		}
 
 		if len(or.Triages) > 0 {
+			// triages need to be included, in case they are not in the cache, in order to show the list on the report
+			if testStats.Regression != nil {
+				testStats.Regression.Triages = or.Triages
+			}
```

Comment:
I think because we need to do that card where we sync the cache primer with the regression tracker, it still happens from time to time that a report has a regression which has not yet been processed by the regression tracker. it would also happen possibly all the time on views which do not use regression tracking.

Reply:
That may still boil back to the need to link up reg tracker and cache primer.